Code/src/main/java/nl/utwente/group10/haskell/env/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Provides an environment for Haskell.
+ */
+package nl.utwente.group10.haskell.env;
```

Comment:
Newline end-of-file missing here and on some other new files. 

(Hint: File > Settings > Editor > General > Other > Ensure line feed at end of file on Save)